articles/email/email-how-use-with-app.md

```diff
@@ -117,10 +125,21 @@ Stunnel can secure SMTP, IMAP4, POP3 (or any other protocol) as follows:
 
 The diagram below shows stunnel encrypting SMTP traffic.
 
-You can get full documentation, how to and FAQ guides from the stunnel website.
-
 ![Stunnel encrypting SMTP traffic](images/email-stunnel.png)
 
+[!NOTE]
```

Comment:
You need a greater than (>) in front of each line in a note.
See: https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html#for-notes